src/clj/game/cards/identities.clj

```diff
@@ -117,6 +117,24 @@
                             :msg "draw a card"
                             :effect (effect (draw 1))}}}
 
+   "Asa Group: Security Through Vigilance"
+   {:events {:corp-install
+             {:delayed-completion true
+              :req (req (and (first-event? state :corp :corp-install)
+                             ;; doesn't fire when installing ice on a central server
```

Comment:
Should still fire on ICE install on central since you can install a _second_ ice protecting that server right?

Reply:
OK, so the inner targeting function needs to disallow selecting an upgrade if the initial target was an ice on a central server.